src/scripts/farming/FarmController.ts

```diff
@@ -105,7 +105,20 @@ class FarmController {
     }
 
     public static mulchAll() {
-        App.game.farming.mulchAll(FarmController.selectedMulch(), this.getAmount());
+        if (!Settings.getSetting('confirmBeformeMulchingAllPlots').observableValue()) {
+            return App.game.farming.mulchAll(FarmController.selectedMulch(), this.getAmount());
+        }
+
+        Notifier.confirm({
+            title: 'Farm Mulch',
+            message: `Are you sure to mulch ${this.getAmount()} of ${MulchType[FarmController.selectedMulch()].replace('_', ' ')} ?`,
```

Comment:
This is not correct english :) My suggestion is:
Title: "Mulch all"
Message: "Are you sure you want to use <amount> <mulch type> on all plots?"
And there is no space between the mulch type and the questionmark

Reply:
Me and english, a great story of love ... almost (more thunder than sunny :smile: )
[34e7529](https://github.com/pokeclicker/pokeclicker/pull/2609/commits/34e7529edc3889768da1afe17b2419d8232ffc3b)